src/__main__.py

```diff
@@ -75,5 +83,7 @@ def main(args: arg.Namespace):
                         help="path to the config file to use for training")
     parser.add_argument("--resume", '-r', type=str, default="", action='store', dest="checkpointfile",
                         help="path to the checkpoint to resume training with")
+    parser.add_argument("--wandb-project", type=str, default="", dest="projectname")
+    parser.add_argument("--run-name", type=str, default="", dest="runname")
```

Comment:
should we add `help` information to each of these arguments as the other ones have? seems self-explanatory enough probably anyways

Reply:
done